tests/foreman/ui/test_contentview.py

```diff
@@ -969,7 +969,9 @@ def test_positive_publish_with_rh_content(session):
 @pytest.mark.skip_if_not_set('fake_manifest')
 @pytest.mark.tier2
 @pytest.mark.skipif((not settings.robottelo.REPOS_HOSTING_URL), reason='Missing repos_hosting_url')
-def test_positive_publish_composite_with_custom_content(session):
+def test_positive_publish_composite_with_custom_content(
+    session, function_entitlement_manifest_org, target_sat
```

Comment:
this test creates a org and uploads a manifest to it, not seeing any use of function_entitlement_manifest_org?